app/assets/stylesheets/pages/_playbook.scss

```diff
@@ -0,0 +1,80 @@
+.pb-bg-navy { background-color: #163963;}
+.pb-footer { background-color: #cedbec; }
+
+.pb-lead {
+  font-size: 1.25rem;
+  line-height: 1.55;
+}
+
+.pb-list {
+  &--bullet {
+    color: $teal;
+
+    a {
+      background: asset-url('playbook/ico-arrow.svg') no-repeat right 8px;
+      background-size: .75rem .75rem;
+      padding-right: 1.25rem;
+    }
+  }
+
+  &--checked li {
+    background: asset-url('playbook/ico-checked.svg') no-repeat left 2px;
+    list-style-type: none;
+    padding-left: 2.25rem;
+  }
+
+  &--checkbox li {
+    background: asset-url('playbook/ico-checkbox.svg') no-repeat left 2px;
+    list-style-type: none;
+    padding-left: 2.25rem;
+  }
+}
+
+.pb-principle-cntnr {
+  background: $white asset-url('playbook/ico-arrow.svg') no-repeat right 2rem center;
+  border-radius: 11px;
+  max-height: 48px;
+  overflow: hidden;
+
+  &:hover {
+    cursor: pointer;
+    text-decoration: none;
+  }
+}
+
+.pb-principle {
+  display: table;
+  font-size: 1.75rem;
+  line-height: .875;
+  table-layout: fixed;
+  width: 100%;
+
+  .table-cell:first-child {
+    width: 3.5rem;
+  }
+}
+
+.pb-nav--top a:hover,
+.pb-nav--top .active, {
+  color: $red;
+  padding-bottom: 10px;
+  position: relative;
+  text-decoration: none;
+
+  &::after {
+    border-bottom: $red 4px solid;
+    border-radius: 12px;
+    content: '';
+    left: 0;
+    position: absolute;
+    top: 100%;
+    width: 100%;
+  }
+}
+
+@media #{$breakpoint-max-sm} {
```

Comment:
we should try to avoid using this - the `breakpoint-max-sm`; it kinda goes against the mobile first principles. instead, if we need diff font sizes, the mobile first font size should be the default and then we can override with larger fonts for larger viewports, and i think we can do this with utility classes (i.e., `h4 sm-h3`)